main.py

```diff
@@ -847,9 +854,27 @@ def escape_appscript(text: str) -> str:
         ")"
     )
 
+    if google_drive_image_link:
+        # Extract image file ID from the passed link
+        file_id_matches = re.match(
```

Comment:
It would be cool if we could also
1) pass a non g-drive image using URLFetchApp ([example](https://developers.google.com/apps-script/reference/forms/image-item))
2) pass image via an attachment to the `!form` command

Though that seems like work for a future commit :^)

Reply:
Indeed, that'd be very fancy.